dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateConstants.java

```diff
@@ -0,0 +1,8 @@
+package com.dotmarketing.portlets.templates.business;
+
+public class TemplateConstants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)